client-protocol.md

```diff
@@ -92,6 +92,13 @@ decryption succeeds, then we're confident that *somebody* used the same
 wormhole code as us. This event pushes the client mood from "lonely" to
 "happy".
 
+Usually the `version` message will be the first one decrypted but it
```

Comment:
Can you please tell me how this may happen? The clients must always process the messages in the correct order, even if they receive them in another. Thus, the `version` phase will always be the first decrypted message.

Reply:
Okay I adjusted this paragraph. Does that help? (I guess I now see why you simply removed it .. maybe this clarification isn't that helpful, but I don't mind being reminded of edge-cases etc so :woman_shrugging: )

Reply:
Yes, the new paragraph is fine for me.